cypress.config.ts

```diff
@@ -24,7 +24,7 @@ export default defineConfig({
     // We've imported your old cypress plugins here.
     // You may want to clean this up later by importing these.
 
-    numTestsKeptInMemory: 0,
+    numTestsKeptInMemory: 2,
```

Comment:
@tejal-shah what do you think about bumping this up to at least 2? 
Or maybe we should just get rid of it?